v1/fleet_units/docker-cleanup.service

```diff
@@ -9,15 +9,16 @@ Restart=on-failure
 RestartSec=20
 TimeoutStartSec=0
 ExecStart=/usr/bin/sh -c ' \
-  if [[ $(/usr/bin/docker ps -a | grep Exited | wc -l) -ne 0 ]]; \
+  if [[ $(/usr/bin/docker ps -a | grep Exited | awk \'{print $1 "\\t" $7 "\\t" $10}\' | grep week | wc -l) -ne 0 ]]; \
   then \
     echo "starting docker cleanup"; \
-    /usr/bin/docker ps -a | grep Exited | awk \'{print $1}\' | xargs /usr/bin/docker rm ; \
+    /usr/bin/docker ps -a | grep Exited | awk \'{print $1 "\\t" $7 "\\t" $10}\' | grep week | awk \'{print $1}\' | head -n25 | xargs /usr/bin/docker rm -v ; \
```

Comment:
can change the output to `starting docker cleanup, 25 max`? (using a new variable, to be defined above)